### PR TITLE
Dev/search choice fixes PRA-173

### PR DESCRIPTION
**Pull request by SvetlanaBeynik**
## One Line Description
Update background and a new card that we want to keep for now.

## Requirements

_What this code should do_

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

## UI/UX Outcome

Screen Shot 2023-08-29 at 10 36 52

_Attach a screenshot here_

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Review by AnnaPestova1 — APPROVED** [review]
looks great!

**Review by ElenaGor8 — APPROVED** [review]
looks nice!

File: src/components/Search/SearchChoice.tsx [hunk 1]
```diff
@@ -37,15 +49,29 @@ const SearchChoice = () => {
...
+          <CardHeader>
+            <Center>
+              <Heading size="md">ADD BY URL </Heading>
+            </Center>
+          </CardHeader>
+          <CardBody>
+            <Text>Discovered a recipe you enjoy online? </Text>
+            <Text>Simply copy and paste the website URL right here.</Text>
+          </CardBody>
+        </Card> */}
```
Comment (rebekahcallkacz) [inline src/components/Search/SearchChoice.tsx:74]: NON-BLOCKING: If this is a "for now" change, this is fine, but, if it's going to be a long time before this feature is added, I might just go ahead and delete it.

**Review by rebekahcallkacz — APPROVED** [review]
This is really nice!

**Pull request merged**